cmd/neofs-node/object.go

```diff
@@ -621,14 +624,27 @@ func (x *fsChainForObjects) IsOwnPublicKey(pubKey []byte) bool {
 func (x *fsChainForObjects) LocalNodeUnderMaintenance() bool { return x.isMaintenance.Load() }
 
 type storageForObjectService struct {
-	local  *engine.StorageEngine
-	putSvc *putsvc.Service
-	keys   *util.KeyStorage
+	cnrs        containercore.Source
+	localEngine *engine.StorageEngine
```

Comment:
`local` was clear enough

Reply:
there was `meta` and it is always local, so IMO it was not

another change cancelled this change anyway